src/test/kotlin/com/example/serverFactoryAPI/ServerFactoryApiApplicationTests.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.serverFactoryAPI
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+
+@SpringBootTest
+class ServerFactoryApiApplicationTests {
+
+	@Test
+	fun contextLoads() {
```

Comment:
@MarkoLazovicCode Is this still in development?

Reply:
It will be resolved together with other tests.